src/Cmfd.cpp

```diff
@@ -1426,12 +1426,12 @@ void Cmfd::setSourceConvergenceThreshold(FP_PRECISION source_thresh) {
 
 
 /**
- * @brief Sets the PolarQuad object in use by the MOC Solver.
- * @param polar_quad A PolarQuad object pointer from the Solver
+ * @brief Sets the Quadrature object in use by the MOC Solver.
+ * @param quadrature A Quadrature object pointer from the Solver
  */
-void Cmfd::setPolarQuadrature(PolarQuad* polar_quad) {
-  _polar_quad = polar_quad;
-  _num_polar = polar_quad->getNumPolarAngles();
+void Cmfd::setQuadrature(Quadrature* quadrature) {
+  _quadrature = quadrature;
+  _num_polar = quadrature->getNumPolarAngles() / 2;
```

Comment:
Well in ``Cmfd`` we always use half the number of polar angles - we never actually use the full number. Maybe I could define a variable like ``_half_num_polar``?